web/src/timerender.ts

```diff
@@ -290,12 +301,20 @@ export function last_seen_status_from_date(
 // List of the dates that need to be updated when the day changes.
 // Each timestamp is represented as a list of length 2:
 //   [id of the span element, Date representing the time]
-type UpdateEntry = {
+type UpdateEntryBase = {
     needs_update: boolean;
-    className: string;
     time: Date;
 };
-let update_list: UpdateEntry[] = [];
+type UpdateEntryWithClassName = UpdateEntryBase & {
```

Comment:
I'm not sure if these changes are worth it, but I feel it's better to proceed with Typescript than without it.